foundation/www/certified_consultants.py

```diff
@@ -0,0 +1,12 @@
+import frappe
+import foundation
+
+no_cache = 1
+
+def get_context(context):
+    '''Returns all certified consultants'''
+    all_consultants = {}
+    all_consultants = frappe.db.sql(""" select website_url,image,name_of_consultant,github_id,discuss_id
+        from `tabCertified Consultant`
```

Comment:
is this custom doctype? Else must be added as JSON

Reply:
No "Certified Consultants" is part of ERPNext now...

https://github.com/frappe/erpnext/pull/14528